symposion/speakers/forms.py

```diff
@@ -17,12 +17,55 @@ class Meta:
             "photo",
             "twitter_username",
             "mobile_number",
+            "interested_mentee",
+            "interested_mentor",
+            "financial_support",
         ]
         help_texts = {
             'mobile_number': _(
                 u"For on-site use only,"
                 u" in case we need to get in touch with you."
             ),
+            'interested_mentee': _(
+                u"We want to support our speakers by connecting them with "
+                u"volunteers from the community to help with the CFP process. "
+                u"If you are interested in receiving mentorship as you "
+                u"prepare your proposal, please select what kind of help "
+                u"you're interested in and you'll hear from us when we have "
+                u"more information! <br/> <b>This is by no means limited to "
+                u"new or first time speakers, everyone is invited to "
+                u"participate!</b>"
+            ),
+            'interested_mentor': _(
+                u"We want to support our speakers by connecting them with "
+                u"volunteers from the community to help with the CFP process. "
+                u"If you are interested in providing mentorship to "
+                u"others as they prepare their proposals, please select what "
+                u"kind of help you're interested in providing and you'll hear "
+                u"from us when we have more information!"
+            ),
+            'financial_support': _(
```

Comment:
@jackiekazil @lorenanicole @jasonamyers new copy updates for financial support text.

Reply:
👍 from me, thanks @ewdurbin!!!!!!!

Reply:
I concur with @lorenanicole :+1: